docs/Changelog.md

```diff
@@ -10529,3 +10529,61 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="TopK-11"></a>**TopK-11**</a>
+
+  Retrieve the top-K largest or smallest elements along a specified axis. Given an input tensor of
+  shape [a_1, a_2, ..., a_n, r] and integer argument k, return two outputs:
+    -Value tensor of shape [a_1, a_2, ..., a_{axis-1}, k, a_{axis+1}, ... a_n]
+      which contains the values of the top k elements along the specified axis
+    -Index tensor of shape [a_1, a_2, ..., a_{axis-1}, k, a_{axis+1}, ... a_n] which
+     contains the indices of the top k elements (original indices from the input
+     tensor).
+  
+  If "largest" is 1 (the default value) then the k largest elements are returned.
+  If "sorted" is 1 (the default value) then the resulting k elements will be sorted.
```

Comment:
Adding `sorted` implies the need of another `Sort` operator. Would you consider split the new top-k into two operators?

Reply:
good question on Sort operator! Actually sort op could be defined as a function op using TopK with 'sorted' attribute. This is one of reasons of this PR.  Hope it clears the question of "split the new top-k into two operators".

Reply:
I feel we start preferring smaller primitives when possible, so I still likely weight `Sort+TopK` more over modifying `TopK`.